Samples/word-citation-management/README.md

```diff
@@ -52,12 +56,21 @@ The sample uses the [@orcid/bibtexParseJs](https://github.com/ORCID/bibtexParseJ
 | Version  | Date | Comments |
 |----------|------|----------|
 | 1.0 | 11-28-2023 | Initial release |
+| 1.1 | 7-2-2025 | Add support for the unified manifest for Microsoft 365 |
+
+## Decide on a version of the manifest
+
+- Add-in only manifest
+  - By default, the sample supports the add-in only manifest, which is the **manifest.xml** file in the sample's root directory **Samples/word-citation-management**. For convenience, a copy of the config files specific to this version can be found in the **manifest-configurations/add-in-only** subfolder.
+  - To run this version, you can skip to the [Run the sample](#run-the-sample) section.
+- Unified manifest for Microsoft 365
+  - To run the unified manifest for Microsoft 365, which is the **manifest.json** file in the sample's root directory **Samples/word-citation-management**, you need to copy the files from the **manifest-configurations/unified** subfolder to the sample's root directory, replacing the existing config files.
```

Comment:
Let's have the manifest.json in the **manifest-configurations/unified** subfolder too instead of the root.